src/IronPythonExtension/IronPythonExtension.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.IO;
+using System.Reflection;
+using Dynamo.Extensions;
+
+namespace IronPythonExtension
+{
+    public class IronPythonExtension : IExtension
```

Comment:
Can you add code documentation to public members at least?

Reply:
will do